core/controllers/moderator_test.py

```diff
@@ -43,20 +47,200 @@ def setUp(self) -> None:
 
         self.save_new_valid_exploration(self.EXP_ID_2, self.user_id)
 
-    def test_unpublished_activities_cannot_be_added_to_featured_list(
+        self.save_new_valid_collection(
+            self.COL_ID_1, self.user_id, exploration_id=self.EXP_ID_1)
+
+        rights_manager.publish_collection(self.user, self.COL_ID_1)
+
+        self.save_new_valid_collection(
+            self.COL_ID_2, self.user_id, exploration_id=self.EXP_ID_1)
+
+    def test_nonexistent_exploration_id(
```

Comment:
What this test case is checking? There is no assert statement.
Also, rephrase the test name with the behaviour that is expected, like:
`test_non_existent_exploration_ids_should_fail` or something around these lines.

Reply:
Ditto for other test cases